setup.py

```diff
@@ -4,7 +4,7 @@
 
 
 def get_version():
-    return '0.0.5'
+    return '0.1.0'
```

Comment:
I considered `0.1.0` instead of `0.0.6` because by using now _usernames_ instead of GitHub's _full-name_ it works the way I had in mind for this feature from the beginning.

Reply:
👌  I agree. Also, this is a breaking change to the previous version ( it doesn't have backward compatibility), it means that it should be 1.0.0, but since we are in an early stage it's fine for me

Reply:
> Also, this is a breaking change to the previous version ( it doesn't have backward compatibility), it means that it should be 1.0.0

That's a good thing to take into consideration, nice. 

In this case, let me know which version would you consider it recommended, `1.0.0` or `0.1.0` and I will go with that one.